src/query/frontend/opencypher/grammar/MemgraphCypherLexer.g4

```diff
@@ -53,6 +53,7 @@ DIRECTORY           : D I R E C T O R Y ;
 DROP                : D R O P ;
 DUMP                : D U M P ;
 DURABILITY          : D U R A B I L I T Y ;
+EDGE_TYPES          : E D G E UNDERSCORE T Y P E S ;
```

Comment:
Where is this used?

Reply:
some old things, I really don't know something happened when ante or me merged some things